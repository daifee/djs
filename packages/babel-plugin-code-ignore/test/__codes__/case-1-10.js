

//@code-ignore-next-statement kuwo
console.log(111)

//    @code-ignore-next-statement kuwo
console.log(222)

//*@code-ignore-next-statement kuwo
console.log(333)

//* @code-ignore-next-statement kuwo
console.log(33333)

//**@code-ignore-next-statement kuwo
console.log(333333333)

/*@code-ignore-next-statement kuwo */
console.log(4)

/*   @code-ignore-next-statement kuwo */
console.log(5)

/**   @code-ignore-next-statement kuwo */
console.log(6)

/**@code-ignore-next-statement kuwo */
console.log(7)