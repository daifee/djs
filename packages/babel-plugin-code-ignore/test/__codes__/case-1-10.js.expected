"use strict";

//

//

//*@code-ignore-next-statement kuwo
console.log(333);

//* @code-ignore-next-statement kuwo
console.log(33333);

//**@code-ignore-next-statement kuwo
console.log(333333333);

/**/

/**   @code-ignore-next-statement kuwo */
console.log(6);

/**@code-ignore-next-statement kuwo */
console.log(7);